### PR TITLE
robo3t: Update to version 1.4.3

### PR DESCRIPTION
**Pull request by b-palaniappan**
Update robo3t to version 1.4.3

### CODE REVIEW

File: bucket/robo3t.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
     "homepage": "https://robomongo.org",
     "license": "GPL-3.0-only",
     "architecture": {
         "64bit": {
-            "url": "https://download.studio3t.com/robomongo/windows/robo3t-1.4.2-windows-x86_64-8650949.zip",
-            "hash": "6f4123c7a1d5ee7121d5f1af942e48140bb6e40008040d4caa17878324e34050",
-            "extract_dir": "robo3t-1.4.2-windows-x86_64-8650949"
+            "url": "https://download.studio3t.com/robomongo/windows/robo3t-1.4.3-windows-x86_64-48f7dfd.zip",
+            "hash": "15C0D1D5751D68023FCCA0F9E0DB14DC69EA4607C2A4C9645DF10B380C1653DD",
+            "extract_dir": "robo3t-1.4.3-windows-x86_64-48f7dfd"
```
Comment (Ash258) [inline bucket/robo3t.json:10]:
```suggestion
            "hash": "15c0d1d5751d68023fcca0f9e0db14dc69ea4607c2a4c9645df10b380c1653dd",
            "extract_dir": "robo3t-1.4.3-windows-x86_64-48f7dfde"
```

**Review by Ash258 — CHANGES_REQUESTED** [review]
Do not work. Database is up and running, but robo3t do not do anything after connection

image

**Comment by b-palaniappan** [comment]
Able to test with my local mongo on Docker.
image

**Comment by pratikpc** [comment]
@b-palaniappan Robo3T seems to now [release](https://github.com/Studio3T/robomongo/releases/latest) via [GitHub](https://github.com/Studio3T/robomongo)

Could you update checkver to use GitHub?
 checkver: github
And switch the download link to the GitHub ones?

**Comment by igitur** [comment]
> @b-palaniappan Robo3T seems to now [release](https://github.com/Studio3T/robomongo/releases/latest) via [GitHub](https://github.com/Studio3T/robomongo)
> 
> Could you update checkver to use GitHub? checkver: github And switch the download link to the GitHub ones?

Fixed in https://github.com/ScoopInstaller/Extras/pull/7196

File: bucket/robo3t.json [hunk 2]
```diff
@@ -1,27 +1,22 @@
...
+    },
     "autoupdate": {
-        "architecture": {
-            "64bit": {
-                "url": "https://download.studio3t.com/robomongo/windows/robo3t-$version-windows-x86_64-$matchRandom.zip",
-                "extract_dir": "robo3t-$version-windows-x86_64-$matchRandom"
-            }
-        }
+        "url": "https://github.com/Studio3T/robomongo/releases/download/v$version/robo3t-$version-windows-x86_64-(?<commit>[a-fA-F0-9]{8}).zip"
     }
```
Comment (pratikpc) [inline bucket/robo3t.json:21]: @b-palaniappan You would need extract DIR here also

File: bucket/robo3t.json [hunk 3]
```diff
@@ -1,27 +1,22 @@
 {
-    "version": "1.4.2",
+    "version": "1.4.3",
     "description": "The free lightweight GUI for MongoDB enthusiasts",
     "homepage": "https://robomongo.org",
     "license": "GPL-3.0-only",
-    "architecture": {
-        "64bit": {
-            "url": "https://download.studio3t.com/robomongo/windows/robo3t-1.4.2-windows-x86_64-8650949.zip",
```
Comment (pratikpc) [inline bucket/robo3t.json:8]: Unless the tool is 32 bit only, for new updates we prefer retaining architecture and 64bit for 64bit apps to minimize inconvenience for 32 bit users

So please use the old format

File: bucket/robo3t.json [hunk 4]
```diff
@@ -1,27 +1,22 @@
...
             "robo3t.exe",
             "Robo 3T"
         ]
     ],
+    "checkver": {
+        "url": "https://github.com/Studio3T/robomongo/releases",
+        "regex": "tag/v([\\w.-]+)"
+    },
     "autoupdate": {
-        "architecture": {
```
Comment (pratikpc) [inline bucket/robo3t.json:20]: Unless the tool is 32 bit only, for new updates we prefer retaining architecture and 64bit for 64bit apps to minimize inconvenience for 32 bit users

So please use the old format

But newer GitHub URLs should be obviously retained

**Comment by pratikpc** [comment]
@igitur we would prefer if you continue to contribute to this specific PR and assist @b-palaniappan instead of creating a new one

**Comment by igitur** [comment]
> @igitur we would prefer if you continue to contribute to this specific PR and assist @b-palaniappan instead of creating a new one

The remote repo for this PR has been deleted. It's really much easier to close this PR and continue on #7196 .

**Pull request closed without merging**